Controller/PlatformUIController.php

```diff
@@ -33,6 +42,28 @@ public function shellAction()
         );
     }
 
+    /**
+     * Load JS or CSS assets.
+     *
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function yuiCombineLoaderAction(Request $request)
```

Comment:
I wouldn't prefix with `yui` as there is nothing specific to YUI, at least code wise.